tremor-cli/src/cli.rs

```diff
@@ -53,6 +56,8 @@ pub(crate) enum Command {
         #[clap( value_parser = clap::value_parser!(String))]
         name: String,
     },
+    /// Pluggable-logging
```

Comment:
I don't think we want to introduce a whole subcommand for logging, this will confuse users as to what it does and provide very little benefit to them. I can totally see that it was helpful in development, but before merging this it should probably be deleted.

Reply:
resolving the comment :D, nicely done!